packages/openapi-v3/src/__tests__/unit/decorators/request-body/request-body.decorator.unit.ts

```diff
@@ -3,11 +3,11 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {post, requestBody, getControllerSpec} from '../../../..';
-import {expect} from '@loopback/testlab';
-import {model, property} from '@loopback/repository';
+import { model, property } from '@loopback/repository';
+import { expect } from '@loopback/testlab';
+import { getControllerSpec, post, requestBody2 } from '../../../..';
```

Comment:
The formatting changes should be reverted.

Reply:
sure. Sorry I didn't make it clear in the PR title, this is a PoC PR, not the real implementation. I should have created this PR as a draft one.